<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsView</class>
 <widget class="QWidget" name="SettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QLabel {
	font-size: 12pt;
}

#mainPage {
	
	background-color: rgb(255, 255, 255);
}

#serialNumber, #softwareVersion, #deviceMainHeader, #deviceConfigHeader {
	font-weight: 900;
	color: #3A6AB4;
}

#deviceMainHeader, #deviceConfigHeader {
	font-size: 14pt;
}

#deviceSerialNumberLabel, #serialNumber {
	background-color: #F2F2F2;
}

#deviceNameLabel, #deviceSoftwareVersionLabel, #softVerWrapper {
	background-color: #F2F2F2; /*#E6E6E6;*/
}

#updateFirmwareBtn {
	border: none;
	background-color: transparent;
	font-size: 12pt;
	text-decoration: underline;
	margin: 0;
	padding: 0;
	font-weight: 900;
	color: #3A6AB4;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="deviceMainHeader">
         <property name="text">
          <string>Основная информация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_17">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="deviceMainInfoGrid" rowminimumheight="30,30,30">
         <item row="0" column="0">
          <widget class="QLabel" name="deviceNameLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Имя устройства</string>
           </property>
           <property name="indent">
            <number>6</number>
           </property>
           <property name="buddy">
            <cstring>name</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">Н/Д</string>
           </property>
           <property name="maxLength">
            <number>50</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="deviceSerialNumberLabel">
           <property name="text">
            <string>Серийный номер</string>
           </property>
           <property name="indent">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="serialNumber">
           <property name="text">
            <string notr="true">Н/Д</string>
           </property>
           <property name="indent">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="deviceSoftwareVersionLabel">
           <property name="text">
            <string>Версия ПО</string>
           </property>
           <property name="indent">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="softVerWrapper" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="softwareVersion">
              <property name="text">
               <string notr="true">Н/Д</string>
              </property>
              <property name="indent">
               <number>6</number>
              </property>
              <property name="buddy">
               <cstring>updateFirmwareBtn</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateFirmwareBtn">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>99</weight>
                <bold>true</bold>
                <underline>true</underline>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>(обновить)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="deviceConfigHeader">
           <property name="text">
            <string>Конфигурация</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveChangesBtn">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_18">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="configTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="deviceBackupLabel_4">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Резервная копия настроек:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createBackupBtn">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Создать</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="restoreBackupBtn">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Восстановить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>saveChanges()</slot>
  <slot>updateFirmware()</slot>
  <slot>createBackup()</slot>
  <slot>restoreBackup()</slot>
  <slot>selectModule(QModelIndex)</slot>
 </slots>
</ui>
